### PR TITLE
 Streamline and simplify collaborator docs

### PR DESCRIPTION
**Pull request by hwillson**
The older `ISSUE_TRIAGE` docs are not only out of date, but don't accurately reflect how Apollo OSS projects are currently maintained. As we bring on new collaborators, it's important that we have a small and direct list of guidelines that collaborators can follow. This list should be kept as small as possible so it's easy to consume, and easy to remember, until the outlined steps become muscle memory for collaborators.

### CODE REVIEW

File: COLLABORATORS.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Thanks for helping make Apollo OSS better! Here are a few quick repo maintenance guidelines intended to help keep Apollo projects happy and healthy.
+
+## Overall
+
+- Please remember to be polite and respectful to all community members, no matter what is being reported, contributed, etc. People who are taking time out of their lives to participate in the Apollo ecosystem should feel welcomed, included and appreciated.
+
+## Issues
+
+- Issues are for bugs only. All other requests should be redirected accordingly, then closed.  Feature requests have their own [repo](https://github.com/apollographql/apollo-feature-requests), and requests for help should go to [Slack](https://www.apollographql.com/slack) or [Stack Overflow](http://stackoverflow.com).
```
Comment (JoviDeCroock) [inline COLLABORATORS.md]: `  Feature` has double space don't know if this is intended.

File: COLLABORATORS.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- If a bug report is valid, add the `confirmed` label, and optionally decide is community help should be requested using the `help-wanted` label. If you’re planning on working on it, assign the issue to yourself.
```
Comment (JoviDeCroock) [inline COLLABORATORS.md]: is community --> if community ?

File: COLLABORATORS.md (hunk 1 above)
Comment (hwillson) [reply COLLABORATORS.md]: Not intended - thanks!

File: COLLABORATORS.md (hunk 2 above)
Comment (hwillson) [reply COLLABORATORS.md]: Great catch - thanks!

**Pull request merged**